_documentation/03-designing-scanner-drivers.md

```diff
@@ -2,5 +2,103 @@
 title: Designing Scanner Drivers
 toc: true
 toc_sticky: true
+h_range: [1,3]
 ---
-This document will contain a complete tutorial as well as information for manufacturers along examples for designing scanner drivers.
\ No newline at end of file
+**Currently PAPPL only provides framework for Printer Application. The codebase is currently under development and will be expanded in a future version of PAPPL to incorporate Multi Function Printer (MFP) functionalities such as scanning.** 
```

Comment:
"... framework for Printer Application." -> "... framework for Printer Application**s**."